addons/account/models/account_move.py

```diff
@@ -3086,6 +3086,10 @@ def _autopost_draft_entries(self):
             ('state', '=', 'draft'),
             ('date', '<=', fields.Date.context_today(self)),
             ('auto_post', '=', True),
+            ('invoice_line_ids', '!=', False),
```

Comment:
This line isn't ok at this place: `('move_type', '=', 'entry')` don't have `invoice_line_ids`.
That would for example exclude all autopost move created for the asset